cockatrice/src/pictureloader.cpp

```diff
@@ -385,11 +384,11 @@ void PictureLoaderWorker::startNextPicDownload()
         picDownloadFailed();
     } else {
         QUrl url(picUrl);
-        QNetworkRequest req(url);
         qDebug() << "PictureLoader: [card: " << cardBeingDownloaded.getCard()->getCorrectedName()
                  << " set: " << cardBeingDownloaded.getSetName()
-                 << "]: Trying to download picture from url:" << url.toDisplayString();
-        networkManager->get(req);
+                 << QString("]: Trying to download picture from %1:").arg(picDownload ? "url" : "cache")
```

Comment:
this is always url, I don't think it's able to know if it's in the cache at this point